python/test/t_GalambosCopula_std.py

```diff
@@ -2,110 +2,109 @@
 
 from __future__ import print_function
 import openturns as ot
+import openturns.testing as ott
 
 ot.TESTPREAMBLE()
 
+ot.RandomGenerator.SetSeed(0)
```

Comment:
nice work!
this line is useless though in a test because it is run independently (contrary to docstrings & examples)